packages/amplify-graphql-conversation-transformer/src/resolvers/assistant-response-stream-pipeline-definition.ts

```diff
@@ -0,0 +1,88 @@
+import { MappingTemplate } from '@aws-amplify/graphql-transformer-core';
+import { ConversationDirectiveConfiguration } from '../conversation-directive-configuration';
+import {
+  createResolverFunctionDefinition,
+  createS3AssetMappingTemplateGenerator,
+  PipelineDefinition,
+  ResolverFunctionDefinition,
+} from './resolver-function-definition';
+import { toUpper } from 'graphql-transformer-common';
+
+/**
+ * The pipeline definition for the assistant response stream mutation resolver.
+ */
+export const assistantResponseStreamPipelineDefinition: PipelineDefinition = {
+  requestSlots: [init(), auth(), verifySessionOwner()],
+  dataSlot: data(),
+  responseSlots: [],
+  field: (config) => ({ typeName: 'Mutation', fieldName: fieldName(config) }),
+};
+
+/**
+ * The init slot for the assistant response mutation resolver.
+ */
+function init(): ResolverFunctionDefinition {
+  return createResolverFunctionDefinition({
+    slotName: 'init',
+    fileName: 'init-resolver-fn.template.js',
+    generateTemplate: (_, code) => MappingTemplate.inlineTemplateFromString(code),
+    substitutions: (_, ctx) => ({
+      GRAPHQL_API_ENDPOINT: ctx.api.graphqlUrl,
+    }),
+  });
+}
+
+/**
+ * The auth slot for the assistant response mutation resolver.
```

Comment:
nit here & throughout: it'd be great to document the important functionality going on in each of the slots rather than reiterating the name. e.g., auth enforces CUP & ownership; session owner verifies session owned by `sub`, etc...